pod-operation/src/components/inverter_board.rs

```diff
@@ -0,0 +1,18 @@
+use rppal::uart::{Parity, Uart};
+pub struct InverterBoard {
+	uart: Uart,
+}
+
+impl InverterBoard {
+	pub fn new() -> Self {
+		let uart = Uart::with_path("/dev/ttyACM0", 9600, Parity::None, 8, 1).unwrap();
```

Comment:
Chore: although the intent is mostly assumed for a standard [8-N-1](https://en.wikipedia.org/wiki/8-N-1), for improved clarity, could we extract each of these arguments to named global constants.